specification/cognitiveservices/data-plane/ComputerVision/stable/v2.0/Ocr.json

```diff
@@ -154,40 +233,127 @@
+          {
+            "$ref": "#/parameters/TextRecognitionMode"
+          }
+        ],
+        "consumes": [
+          "application/octet-stream"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "202": {
+            "description": "The service has accepted the request and will start processing later.",
+            "headers": {
+              "Operation-Location": {
+                "description": "URL to query for status of the operation. The operation ID will expire in 48 hours. ",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "#/definitions/ComputerVisionError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Domain Model analysis request": {
+            "$ref": "./examples/SuccessfulBatchReadFileWithStream.json"
+          }
+        }
+      }
     }
   },
   "definitions": {
     "TextOperationResult": {
       "type": "object",
       "properties": {
         "status": {
-          "type": "string",
           "description": "Status of the text operation.",
-          "enum": [
-            "Not Started",
-            "Running",
-            "Failed",
-            "Succeeded"
-          ],
-          "x-ms-enum": {
-            "name": "TextOperationStatusCodes",
-            "modelAsString": false
-          },
-          "x-nullable": false
+          "$ref": "#/definitions/Status"
         },
         "recognitionResult": {
-          "$ref": "#/definitions/RecognitionResult"
+          "$ref": "#/definitions/TextRecognitionResult"
+        }
+      }
+    },
+    "ReadOperationResult": {
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "Status of the text operation.",
+          "$ref": "#/definitions/Status"
+        },
+        "recognitionResults": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TextRecognitionResult"
+          }
         }
       }
     },
-    "RecognitionResult": {
+    "Status": {
+      "type": "string",
+      "enum": [
+        "Not Started",
+        "Running",
+        "Failed",
+        "Succeeded"
+      ],
+      "x-ms-enum": {
+        "name": "TextOperationStatusCodes",
+        "modelAsString": false
+      },
+      "x-nullable": false
+    },
+    "TextRecognitionResult": {
       "type": "object",
+      "required": [
+        "lines"
+      ],
       "properties": {
         "lines": {
           "type": "array",
           "items": {
             "$ref": "#/definitions/Line"
           }
+        },
+        "page": {
+          "type": "integer"
+        },
+        "width": {
+          "type": "number"
+        },
+        "height": {
+          "type": "number"
+        },
+        "clockwiseOrientation": {
+          "type": "number"
```

Comment:
Please add descriptions for each of these properties. Is the page 0-based or 1-based? Are the width and height using the unit property below? IS clockwiseOrientation in radians or degrees?

Reply:
Thanks! I have updated the properties based on the [API documentation](https://ocrv1-usw2-dev.portal.azure-api.net/docs/services/5adf991815e1060e6355ad44/operations/5be108e7498a4f9ed20bf96d)